app/src/main/java/com/better/alarm/persistance/AlarmDatabaseHelper.java

```diff
@@ -43,27 +43,25 @@ public AlarmDatabaseHelper(Context context, Logger log) {
   @Override
   public void onCreate(SQLiteDatabase db) {
     // @formatter:off
-    db.execSQL(
-        "CREATE TABLE alarms ("
-            + "_id INTEGER PRIMARY KEY,"
-            + "hour INTEGER, "
-            + "minutes INTEGER, "
-            + "daysofweek INTEGER, "
-            + "alarmtime INTEGER, "
-            + "enabled INTEGER, "
-            + "vibrate INTEGER, "
-            + "message TEXT, "
-            + "alert TEXT, "
-            + "prealarm INTEGER, "
-            + "state STRING);");
+    db.execSQL("CREATE TABLE alarms (" +
+            "_id INTEGER PRIMARY KEY," +
+            "hour INTEGER, " +
+            "minutes INTEGER, " +
+            "daysofweek INTEGER, " +
+            "alarmtime INTEGER, " +
+            "enabled INTEGER, " +
+            "vibrate INTEGER, " +
+            "message TEXT, " +
+            "alert TEXT, " +
+            "prealarm INTEGER, " +
+            "state STRING, " +
+            "deleteondismiss INTEGER);");
```

Comment:
How about using repeat column with value -1 for this? This will not require changing the database layout. In the future I wanted to get rid of the database and replace it with a protobuf datastore, but this is not yet implemented.